recipes/vulkan-loader/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+
+#include <vulkan/vulkan.hpp>
+
+int main()
+{
+    vk::ApplicationInfo appInfo;
+    appInfo.pApplicationName = "ConanTestApp";
+    appInfo.pEngineName = "ConanTestEngine";
+    appInfo.apiVersion = VK_API_VERSION_1_2;
```

Comment:
Not that it matters too much but is requiring vulkan 1.2 a big requirement? Wouldn't 1.0 suffice?